packages/xstate-react/src/useActor.ts

```diff
@@ -31,11 +31,7 @@ const noop = () => {
 function defaultGetSnapshot<TEmitted>(
   actorRef: ActorRef<any, TEmitted>
 ): TEmitted | undefined {
-  return isActorWithState(actorRef)
-    ? actorRef.state
-    : 'getSnapshot' in actorRef
-    ? actorRef.getSnapshot()
-    : undefined;
+  return 'getSnapshot' in actorRef ? actorRef.getSnapshot() : undefined;
```

Comment:
similarly here - we can't remove accessing `state` because it's a breaking change. A user should be free to update `@xstate/react` without updating `xstate` at the same time (and vice versa)

Reply:
Done in 3be0918b94ea24074fd63e2697f2c9b72e74d815